apps/web/src/components/Dropdown/Dropdown.tsx

```diff
@@ -116,25 +124,28 @@ export const Dropdown = ({
           title={title}
           type="button"
           size={size}
-          aria-expanded="false"
+          aria-expanded={isOpen ? 'true' : 'false'}
```

Comment:
Totalement facultatif :
```suggestion
          aria-expanded={`${isOpen}`}
```